en/FlyView/FlyView.md

```diff
@@ -1,79 +1,328 @@
-The default map provider is Bing since it seems to provide better Hybrid maps. 
-You can change the map provider from the General page of Settings.
+The default page displays vehicle telemetry - use the drop down menu on the to right to select the other options.
 
-## Video
+### Values (Telemetry)
 
-At the lower left of the display you will see video output. 
-QGroundControl supports RTP and RTSP video streaming over your vehicles UDP connection. 
-It also support directly connected UVC device support. 
-More details on QGC Video support can be found on the [Video README](https://github.com/mavlink/qgroundcontrol/blob/master/src/VideoStreaming/README.md).
+The values page shows telemetry information; by default the altitude (relative to the home location) and the ground speed.
 
-By clicking on the video you can make it be the main display for the Fly view.
+![Instrument Page - for values/telemetry](../../assets/fly/instrument_page_values.jpg)
 
-## Instrument Panel
+You can configure what information is display by clicking the small gear icon on the top left of the panel.
+Each value can be displayed in normal or "large" size (large size shows just one value per row in the page, while normal shows 2).
 
-To the right is an instrument panel showing you current information on your vehicle. 
-The center section of the panel has multiple pages. 
-You can switch between pages by clicking on the center section. 
+![Instrument Page - values settings](../../assets/fly/instrument_page_values_settings.jpg)
 
-### Telemetry page
 
-![Instrument Telemetry Page](../../assets/fly/InstrumentTelemetryPage.jpg)
+### Camera
 
-The values shown within the telemetry page can be configured by clicking on the small gear icon.
+The camera page is used to configure and control the camera.
+For a camera connected directly to the Flight Controller the only available option is camera triggering:
 
-### Vehicle Health page
+![Instrument Page - for Camera](../../assets/fly/instrument_page_camera.jpg)
 
-![Instrument Health Page](../../assets/fly/InstrumentHealthPage.jpg)
+When connected to camera that supports the [MAVLink Camera Protocol](https://mavlink.io/en/services/camera.html) you can additionally configure and use other camera services that it makes available.
+For example, if your camera supports video mode you will be able to  switch between still image capture and video mode, and start/stop recording.
 
-This page shows you the health of the systems within your Vehicle.
-If any of the systems switches from healthy to unhealthy this page will automatically be switched to.
+![Instrument Page - Camera MAVLink Settings](../../assets/fly/instrument_page_camera_mavlink.jpg)
 
-### Vibration Clipping page
+Advanced settings can be changed by clicking the gear icon at the top left of the page.
+The settings that are displayed will depend on/are a property of the camera.
```

Comment:
@dogmaphobic In FlyView, if you're using the MAVLink camera you get option to configure settings. Are these dynamic based on the config file, or hard coded? (I have said here dynamic)

Reply:
Most settings come dynamically from the camera. That's what comes in the _camera definition file_ (the XML file). There are a few hardcoded, common settings at the bottom:

- Photo Mode (Single/Time Lapse)
- Photo Interval (if Time Lapse)
- Reset Camera Defaults (sends a reset command to the camera)
- Format (storage)

Reply:
Thanks!